CoreSchemaView/release/modules/ext/docs/CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/schemaview-toc.xml

```diff
@@ -2,13 +2,15 @@
 <!DOCTYPE toc PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp TOC Version 2.0//EN" "http://java.sun.com/products/javahelp/toc_2_0.dtd">
 <toc version="2.0">
     <tocitem text="Views" mergetype="javax.help.SortMerge">
-        <tocitem text="Schema View">
+        <tocitem text="Schema View" mergetype="javax.help.SortMerge">
+            <tocitem text="Schemas" target="au.gov.asd.tac.constellation.graph.schema.Schema"/>
```

Comment:
To follow up your comment regarding location, I think this is better placed in Graph > Graph Overview (alongside Types and Attributes)